### PR TITLE
Fixes segfault on client/cpp/cli.cpp and corrects path in scripts/run-tests.sh

### PR DESCRIPTION
**Pull request by WildCryptoFox**

### CODE REVIEW

File: client/cpp/cli.cpp [hunk 1]
```diff
@@ -37,6 +37,10 @@ void handle_request(KvsClientInterface *client, string input) {
   vector<string> v;
   split(input, ' ', v);
 
+  if (v.size() == 0) {
+    std::exit(EXIT_SUCCESS);
+  }
+
   if (v[0] == "GET") {
```
Comment (WildCryptoFox) [inline client/cpp/cli.cpp:44]: The vector has length zero when the input does not contain a space. `v[0]` triggers the segfault.

**Comment by vsreekanti** [comment]
Thanks, @WildCryptoFox!

**Comment by WildCryptoFox** [comment]
@vsreekanti I think you merged in the wrong direction. You added a merge commit into this PR but didn't merge this into master. Never seen this one before!

**Comment by vsreekanti** [comment]
@WildCryptoFox I merged master into this branch because there were new commits on master and the branch had to be updated. I had to wait for Travis to run before merging this back into master but forgot to check up again. Either way, looks like we're all set. Thanks again for your PR!

**Pull request merged**